File: server/Models/Req/LoginReq.cs
using System.ComponentModel.DataAnnotations;

namespace ConnectoDb.Server.Models.Req;

public class LoginReq
{
    [Required, MinLength(3), MaxLength(256)]
    public string Username { get; set; } = null!;

    [Required, MinLength(8), MaxLength(256)]
    public string Password { get; set; } = null!;
}

File: clients/java/src/main/java/com/connecto/client/models/LoginReq.java
package com.connecto.client.models;

import com.google.gson.annotations.SerializedName;

public class LoginReq {
    public String username;
    public String password;

    public LoginReq(String username, String password) {
        this.username = username;
        this.password = password;
    }
}
